source/about/accessibile_html_error_messages.md

```diff
@@ -1,58 +1,58 @@
 # Error messages in HTML papers
 
-The process of converting our papers to HTML is a work in progress and as we improve the process you will see fewer errors and formatting issues. 
+To produce accessible HTML papers we use a LaTeXML converter created and maintained by National Library of Standards and Technology (NIST).  
 { .intro }
 
-The errors that you have seen when viewing  papers in HTML after submission or on arXiv.org are caused by our inability to translate certain Tex and LaTex software packages into HTML. TeX and LaTeX, pronounced ‘tek’ and ‘LAH-tek’, are open source software packages used to accurately display complex mathematical formulae, symbols and tables in research papers. Due to the open source nature of TeX and LaTeX and its popularity, there are an inordinate number of packages available to choose from. This makes supporting all of them very difficult. We are continuously reviewing which features and packages are most popular in an effort to provide error free, accessible HTML papers.
+The errors that you have seen when viewing papers in HTML after submission or on arXiv.org are caused when the LaTeXML converter is unable to translate certain TeX and LaTeX software packages. TeX and LaTeX, pronounced ‘tek’ and ‘LAH-tek’, are open source software packages used to accurately display complex mathematical formulae, symbols and tables in research papers. Due to the open source nature of TeX and LaTeX and its popularity, there are many packages available to choose from. This makes supporting all of them very difficult. NIST is routinely assessing and adding LaTeX packages in an effort to provide greater versatility to the LaTeXML converter.
 
-Please note: These errors are only the result of our  inability to properly translate and display certain TeX and LaTex packages as HTML. These errors are not an indication of a larger system failure.
 
-## HTML errors readers may encounter
+## HTML errors you may see:
 
 When browsing and reading HTML papers on arXiv.org, you may see formatting errors in red like below: 
 
-![HTML LaTeX processing error for online id](images/reader-error-01.png)
+![HTML LaTeX processing error for online id](images/reader-error-01.png){.mkd-img-border}
```

Comment:
This an the next two images need alt text

Reply:
I have added alt text.